Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/ui/certificateselection/DccTicketingCertificateSelectionViewModel.kt

```diff
@@ -72,7 +74,11 @@ class DccTicketingCertificateSelectionViewModel @AssistedInject constructor(
         }
 
     fun onUserCancel() {
-        events.postValue(ShowCancelConfirmationDialog)
+        if (hasValidCertificate) {
```

Comment:
`items.value?.isNotEmpty()` ? and no need for `hasValidCertificate`

Reply:
I'm not checking `items` but `certificates`

Reply:
Ah Ok, please make hasValidCertificate `private` ? it is only used in the VM

Reply:
updated